README.md

```diff
@@ -1 +1,43 @@
-# first-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# First Project
+
+> This is the initial project in Microverse. The aim is to enable me to write professional work and have gitflow best practices.
+
+## Built With
+
+- Html
+- Css
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
```

Comment:
> # [Optional]
> Good job setting up this README file.
> If check the [Microverse template](https://github.com/microverseinc/readme-template) you will see there is a section where you're supposed to add `Setup`.
> So you need to add this project set up so that any non-technical user can set up your app on a local machine.
> 
> Example:
> 
> * git clone `your repo`
> * cd into the folder
> * open `index.html` with live server
> 
> Kindly update the README file before merging, 👀 on you.

I have updated the README file